app.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const app = express();
+const {
+getTopics
+} = require('./controllers/topics-controllers');
+
+
+app.use(express.json());
```

Comment:
Is this needed yet as we are not dealing with anything from a request?

Reply:
test is not passing without this :)